src/app/containers/list/list.service.ts

```diff
@@ -1,14 +1,27 @@
 export class ListContainersService {
 
+  /**
+   * This gets the docker pull command
+   *
+   * @param {string} path The path of the tool (quay.io/namespace/toolname)
+   * @param {string} [tagName=''] The specific version of the docker image to get
+   * @returns {string} The docker pull command
+   * @memberof ListContainersService
+   */
   getDockerPullCmd(path: string, tagName: string = ''): string {
     let dockerPullCmd = 'docker pull ';
+
+    // The below handles docker hub registries
     const prefix = 'registry.hub.docker.com/';
     if (path.indexOf(prefix) !== -1) {
       path = path.replace(prefix, '');
     }
+
+    // Not sure what this handles
     if (path.indexOf('_/') !== -1) {
```

Comment:
Looks like its killing underscores. Not sure why

Reply:
This is in the original code, I'm not sure why it's needed.